installutils/helminstall/client.go

```diff
@@ -38,70 +38,73 @@ type HelmClient interface {
 }
 
 // an interface around Helm's action.Install struct
-type HelmInstall interface {
+type HelmInstaller interface {
 	Run(chrt *chart.Chart, vals map[string]interface{}) (*release.Release, error)
 }
 
 // an interface around Helm's action.Uninstall struct
-type HelmUninstall interface {
+type HelmUninstaller interface {
 	Run(name string) (*release.UninstallReleaseResponse, error)
 }
 
-var _ HelmInstall = &action.Install{}
-var _ HelmUninstall = &action.Uninstall{}
+var _ HelmInstaller = &action.Install{}
+var _ HelmUninstaller = &action.Uninstall{}
 
 // interface around needed afero functions
-type TempFile interface {
-	NewTempFile(fs afero.Fs, dir, prefix string) (f afero.File, err error)
-	WriteFile(fs afero.Fs, filename string, data []byte, perm os.FileMode) error
+type Fs interface {
```

Comment:
nit: can we call this `FsExtended` or something related to indicate that it represents fs helper functions, rather than naming it the same as the `afero.Fs`

Reply:
good point, I'll use `FsHelper`